src/main/jbake/content/faq.ad

```diff
@@ -38,11 +38,11 @@ https://stackoverflow.com/questions/tagged/opennlp[forums].
   <dd>To train the name finder model you need training data that contains the entities you would
   like to detect.
 Have a look at our manual, in special the sections under the
-<a href="/docs/1.7.2/manual/opennlp.html#tools.namefind.training">Name Finder Training API</a>.
+<a href="/docs/2.1.1/manual/opennlp.html#tools.namefind.training">Name Finder Training API</a>.
 At the beginning of that section you can see how the data has to be marked up. Please note you that you need many sentences to successfully train the name finder.</dd>
 
   <dt>How can I speed up my MaxEnt training time</dt>
-  <dd>Try tweaking the value of <a href="/docs/1.7.2/apidocs/opennlp-tools/opennlp/tools/util/TrainingParameters.html#THREADS_PARAM">TrainingParameters.THREADS_PARAM</a>.</dd>
+  <dd>Try tweaking the value of <a href="/docs/2.1.1/apidocs/opennlp-tools/opennlp/tools/util/TrainingParameters.html#THREADS_PARAM">TrainingParameters.THREADS_PARAM</a>.</dd>
```

Comment:
I wonder if we could automate it, or maybe create a release checklist document that mentions what needs to be updated where, so that it doesn't get outdated soon again.